src/App/Containers/Layout/footer.jsx

```diff
@@ -10,7 +10,9 @@ import {
     NetworkStatus,
     ToggleFullScreen,
     TogglePositions,
-    ToggleSettings }                  from '../../Components/Layout/Footer';
+}                  from '../../Components/Layout/Footer';
```

Comment:
@easteregg Please align `from` statement and move the curly braces to the upper line.